pkg/service/types.go

```diff
@@ -18,6 +18,10 @@ type MeshService struct {
 	Name string
 }
 
+func (ms MeshService) IsLocal() bool {
+	return true
```

Comment:
Why is defaulting to true, shouldn't there be some logic to figure it out from mesh service

Reply:
Placeholder until we implement TODO: Implement this. Do an exact match for local on .ClusterDomain

Reply:
tim can you add a TODO comment on this?